tests/init/neg/closureLeak.check

```diff
@@ -1,16 +1,14 @@
 -- Error: tests/init/neg/closureLeak.scala:11:14 -----------------------------------------------------------------------
 11 |    l.foreach(a => a.addX(this)) // error
    |              ^^^^^^^^^^^^^^^^^
-   |              Cannot prove the method argument is hot. Only hot values are safe to leak.
-   |              Found = Fun { this = ThisRef[class Outer], owner = class Outer }. Calling trace:
-   |              -> class Outer {	[ closureLeak.scala:1 ]
-   |                 ^
-   |              -> l.foreach(a => a.addX(this)) // error	[ closureLeak.scala:11 ]
-   |                           ^^^^^^^^^^^^^^^^^
+   |Could not verify that the method argument is transitively initialized (Hot). It was found to be a function where "this" is (the original object of type (class Outer) where initialization checking started) and the function owner is an object of type (class Outer). Only transitively initialized arguments may be passed to methods (except constructors). Calling trace:
```

Comment:
I'm sorry to hold this up but I have one more question for @liufengyun : In the message that we give for a `Function`, is the explanation of `this` and `owner` accurate and as helpful as it can be?

Reply:
Remove the function owner from output but keep the value of "this".